imsim/stamp.py

```diff
@@ -629,6 +629,12 @@ def draw(self, prof, image, method, offset, config, base, logger):
             # don't draw anything.
             return image
 
+        my_bounds = image.bounds & base['current_image'].bounds
+        if my_bounds.area() == 0:
+            # The stamp for this object doesn't overlap with the
+            # current image, so skip it.
+            return image
```

Comment:
Does this ever happen?  I thought the galsim config code already checked for this.

Reply:
Yes, it does happen.   I saw right away it while trying to debug the original issue.  Since we are selecting a 100 pixel region around the edge of the CCD for bright stars not actually landing on the CCD, the faint ones like this also get included.

Reply:
Is  GalSim #1258 going to be part of a release soon?  If so, I would just as soon omit the stamp-related changes and only include Chris' config changes in this PR.

Reply:
It's already released as part of 2.5.1.